### ISSUE TITLE
Ability to upload game directly from TIC/SURF

### ISSUE THREAD

**Issue by darkhog** [nesbox/TIC-80#534]
Perhaps adding ability to login/authenticate via the app and then upload carts by the standard directory controls. The way it would work is that an additional network resource, let's call it [tic.computer/user] would be mounted, similarly to [tic.computer/play], except [tic.computer/user] would be writable and when writing to it, the cart would be uploaded to the site and listed under the user's account.

**Comment by trelemar** [comment]
I agree 100%. There's a few different ways this could be implemented. (I searched here to see if this has been opened before, you beat me to it :P)

Imagine a github-like update system for different game carts. The game it self would just have to store it's own cart id number or something. When you type `push cart` or whatever the command is called it would check the cart's id with the one on tic.computer/play and ask for your username and password. ALL in the console it's self.

I want this so bad. It would bring life to the WIP section as well. I'd use it all the time.
